src/quo2/components/gradient/gradient_cover/style.cljs

```diff
@@ -0,0 +1,5 @@
+(ns quo2.components.gradient.gradient-cover.style)
+
+(def root-container
+  {:flex   1
+   :height 252})
```

Comment:
is this height always the same? I would have thought `flex` would be more applicable or a height that we can set? 🤔

Reply:
Yes, the height is always the same. `flex: 1` is a bit unnecessary. I will remove it. 👍